modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -162,11 +165,25 @@ To create a custom asset:
 . In Exchange, select *Publish new asset*.
 . Enter the asset portal name.
 . Select the asset type *Custom* from the drop-down list.
-. (Optional) To share a file with users, choose the file.
-. (Optional) Select *Advanced* and edit the advanced settings: *GroupId*, *AssetId*, and *Version*. Exchange generates the group ID, asset ID, and version for you, and you can change these values here. The advanced settings are most often used to change the asset version.
-. (Optional) Select an option for *Lifecycle state*. The default is *Stable*.
+. Click *Choose the file* to select the file to upload.
+. To edit the *GroupId*, *AssetId*, and *Version*, click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values here. The advanced settings are most often used to change the asset version.
+. Select an option for *Lifecycle state*. The default is *Stable*.
 . Select *Publish*.
 
+== Create a Ruleset Asset
+Rulesets are a group of rules or guidelines that define an application, for example best practices or industry-specific standards. Add a zip file with multiple YAML files that describe the new ruleset.
+
+. In Exchange, select *Publish new asset*.
+. Enter the asset portal name.
+. Select the asset type *Ruleset* from the drop-down list.
```

Comment:
```suggestion
. From the drop-down list, select the asset type *Ruleset* .
```
This is just my particular thing 🤣  --I'm always going to put the preposition/navigation first